codegen/templates/workflow.tmpl

```diff
@@ -133,10 +134,24 @@ func (w {{$workflowStruct}}) Handle(
 	{{end}}
 
 	clientHeaders := map[string]string{}
+	{{if (ne (len $defaultHeaders) 0) }}
+	var ok bool
+	var h string
+	{{range $i, $k := $defaultHeaders}}
+	k := textproto.CanonicalMIMEHeaderKey("{{$k}}")
+	h, ok = reqHeaders.Get(k)
+	if ok {
+		clientHeaders[k] = h
+	}
```

Comment:
nit:  a more go-idiomatic way would have been to do something like
```
if h, ok := reqHeaders.Get(k); ok {
  ...
}
```
That would limit the variable scope

Reply:
either way, it's not a big deal but what we have prevents creating several new variables and is intentional.